app/server/api/filter_api.py

```diff
@@ -51,11 +51,12 @@ def post(self):
 
         create_filter = SavedFilter(
             name=filter_name,
-            filter=filter_attributes
+            filter=filter_attributes,
+            organisation_id=g.user.default_organisation_id
         )
 
         db.session.add(create_filter)
-        db.session.commit()
+        db.session.flush()
```

Comment:
At this point, i feel like there's not a lot of reason to have the commit in the after_request hook :|

Reply:
i still like it